lib/indexer.rb

```diff
@@ -98,18 +101,25 @@ def find_cdxj_files
   end
 
   def write_cdxj(warc_file)
-    cdxj_path = cdxj_path(warc_file)
-    cdxj_path.parent.mkpath unless cdxj_path.parent.directory?
-
-    cmd = ['cdxj-indexer', '--post-append', '--output', cdxj_path.to_s, warc_file.to_s]
-    @log.info("generating cdxj #{cdxj_path} for #{warc_file}")
-    @log.error("cdxj error while parsing #{warc}") unless system(*cmd)
+    cdxj = cdxj_path(warc_file)
```

Comment:
To avoid confusion I renamed `cdxj_path` to `cdxj`, since there is a method named `cdxj_path`.